.github/workflows/CI.yaml

```diff
@@ -80,3 +80,15 @@ jobs:
     - name: Check Image Size
       run: |
         make sizecheck
+  check-spelling:
+    name: Check Spelling
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code.
+        uses: actions/checkout@v1
+      - name: misspell
+        uses: reviewdog/action-misspell@v1
+        with:
+          github_token: ${{ secrets.github_token }}
```

Comment:
We do not need to provide a token here, `github.token` is used by default and should be sufficient.